scalafmt-core/shared/src/main/scala/org/scalafmt/config/DanglingParentheses.scala

```diff
@@ -5,6 +5,7 @@ import metaconfig._
 case class DanglingParentheses(
     callSite: Boolean,
     defnSite: Boolean,
+    kwSite: Boolean = false,
```

Comment:
What does "kw" stand for? Can we use a more self-descriptive name?

Reply:
well, it stands for `keyword`. but since keywords cover a lot more, perhaps it should be `ctrl` (for control flow, which is exactly `if`, `while` and `for`... well, and `case` but it doesn't apply here).